nms/app/packages/magmalte/server/network/routes.js

```diff
@@ -0,0 +1,272 @@
+            ...commonField,
+            cellular: {
+              ...DEFAULT_CELLULAR_CONFIG,
+              feg_network_id: data.fegNetworkID,
+            },
+            dns: DEFAULT_DNS_CONFIG,
+            federation: {feg_network_id: data.fegNetworkID},
+          },
+        });
+      } else if (data.networkType === CWF) {
+        resp = await MagmaV1API.postCwf({
+          cwfNetwork: {
+            ...commonField,
+            dns: DEFAULT_DNS_CONFIG,
+            federation: {feg_network_id: data.fegNetworkID},
+            carrier_wifi: {
+              aaa_server: {
+                accounting_enabled: true,
+                create_session_on_auth: true,
+                idle_session_timeout_ms: 500000,
+              },
+              default_rule_id: '',
+              eap_aka: {},
+              network_services: ['policy_enforcement', 'dpi'],
+            },
+          },
+        });
+      } else if (data.networkType === XWFM) {
+        resp = await MagmaV1API.postCwf({
+          cwfNetwork: {
+            ...commonField,
+            dns: DEFAULT_DNS_CONFIG,
+            federation: {feg_network_id: data.fegNetworkID},
+            carrier_wifi: {
+              aaa_server: {
+                accounting_enabled: true,
+                create_session_on_auth: true,
+                idle_session_timeout_ms: 500000,
+              },
+              default_rule_id: '',
+              eap_aka: {},
+              network_services: [],
+            },
+          },
+        });
+      } else if (data.networkType === FEG) {
+        resp = await MagmaV1API.postFeg({
+          fegNetwork: {
+            ...commonField,
+            dns: DEFAULT_DNS_CONFIG,
+            federation: {
+              aaa_server: {},
+              csfb: {},
+              eap_aka: {},
+              gx: {},
+              gy: {},
+              health: {},
+              hss: {},
+              s6a: {},
+              served_network_ids: data.servedNetworkIDs.split(','),
+              swx: {},
+            },
+          },
+        });
+      } else if (data.networkType === SYMPHONY) {
+        resp = await MagmaV1API.postSymphony({
+          symphonyNetwork: {
+            ...commonField,
+          },
+        });
+      } else {
+        await MagmaV1API.postNetworks({
+          network: {
+            ...commonField,
+            type: data.networkType,
+            dns: DEFAULT_DNS_CONFIG,
+          },
+        });
+      }
+
+      MagmaV1API.postNetworksByNetworkIdTiers({
+        networkId: networkID,
+        tier: DEFAULT_UPGRADE_TIER,
+      });
+
+      // Add network to organization
+      if (req.organization) {
+        const organization = await req.organization();
+        const networkIDs = [...organization.networkIDs, networkID];
+        await organization.update({networkIDs});
+      }
+    } catch (e) {
+      logger.error(e, {
+        response: e.response?.data,
+      });
+      res
+        .status(200)
```

Comment:
200 for an error?

Reply:
Yeah. Not sure. what is going on here. They have marked the status as 200,  However the success is marked as false
        .status(200)
        .send({
          success: false, <===
          message: e.response?.data.message || e.toString(),
          apiResponse: e.response?.data,
        })

I have to understand what's going on here.